README.md

```diff
@@ -5,9 +5,23 @@ This is a lab for investigating adding a layer of webservices to Joomla
 
 For more details join #webservices channel at https://glip.com/
 
-
 How to install
 ====
+* Install a full current Joomla somewhere
+* git clone https://github.com/joomla-projects/webservices.git OR https://github.com/chrisdavenport/webservices.git
+* Zip or tar-gzip everything in the /component directory
+* Install component package file as an extension in the usual way
+* Copy all the files from the github clone to the web root of the CMS
+* sudo composer require willdurand/Negotiation (don't know why this is needed, but it is)
```

Comment:
You haven't updated the `composer.lock` file in this PR (I don't know if that's intentional or not) - that's the problem